ios/TPSTwitterModule/TPSTwitterModule.m

```diff
@@ -8,6 +8,15 @@
 typedef void(^TWTRSessionHandler)(TWTRSession *session, NSError *error);
 static NSString *const kTWTConsumerKey = @"consumerKey";
 static NSString *const kTWTConsumerSecret = @"consumerSecret";
+static NSString *const kTPSTwitterErrorDomain = @"com.tipsi.twitter";
+
+typedef NS_ENUM(NSUInteger, TPSTwitterError) {
+    TPSTwitterErrorNoConsumerKey,
+    TPSTwitterErrorNoConsumerSecret,
+    TPSTwitterErrorNoTwitterKeys,
+    TPSTwitterErrorNoAuthConfiguration
+    
+};
```

Comment:
Looks good. 
Last thing, please check exporting constants section (https://facebook.github.io/react-native/docs/native-modules-ios.html#exporting-constants) I think you should export error domain and error status codes.